packages/design-tokens/src/tokens/functional/components/pricing-cards/pricing-cards.json

```diff
@@ -0,0 +1,42 @@
+{
+  "brand": {
+    "PricingOptions": {
+      "items": {
+        "gap": {
+          "value": "var(--base-size-24)"
+        },
+        "container": {
+          "padding": {
+            "inline": {
+              "value": "0"
+            }
+          }
+        }
+      },
+      "items-2": {
```

Comment:
I'm fine with this for now, but with the recent v8 naming conventions is this okay to use kebab case? should it be `items2`?